src/main/java/org/hiero/sketch/spreadsheet/QuantileSketch.java

```diff
@@ -0,0 +1,253 @@
+package org.hiero.sketch.spreadsheet;
+
+import org.checkerframework.checker.nullness.qual.NonNull;
+import org.hiero.sketch.dataset.api.ISketch;
+import org.hiero.sketch.dataset.api.PartialResult;
+import org.hiero.sketch.table.*;
+import org.hiero.sketch.table.api.IColumn;
+import org.hiero.sketch.table.api.IMembershipSet;
+import org.hiero.sketch.table.api.IRowOrder;
+import org.hiero.sketch.table.api.IndexComparator;
+import rx.Observable;
+
+import java.security.InvalidParameterException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class QuantileSketch implements ISketch<Table, Table> {
+
+    private final List<OrderedColumn> sortOrder;
+
```

Comment:
too many empty lines
Please define a separate class ColumnSetOrdering which contains a list of OrderedColumn.